searx/engines/google_images.py

```diff
@@ -72,11 +72,26 @@ def scrap_img_by_id(script, data_id):
     """Get full image URL by data-id in parent element"""
     img_url = ''
     _script = script.split('\n')
```

Comment:
The problem of the old (and new) implementation is that there are no longer any `\n` in the result from google (the length of _split is always one).

Reply:
So there is no need for the .split() Method? Or would you suggest another approach?

Reply:
> So there is no need for the .split() Method? Or would you suggest another approach?

I implemented an alternative solution in my branch [fix-909-mhe](https://github.com/return42/searxng/commits/fix-909-mhe) .. would you please have a look / see commit

Reply:
@return42 I think your solution is much cleaner because of the regex.

How should we proceed?

Reply:
If it is OK for you, you can reset you branch for this PR by my branch. The alternative is I open another PR, but IMO it is better we reset this branch to have a consistent history in github's PR ..

Reply:
sorry messed something up. will fix it now